content/backmatter.tex

```diff
@@ -829,72 +873,72 @@ \section{Version 1.4}
 \ChangelogRef{dep:overview, dep:rationale}%
 %
 \item Deprecated header directory \HEADER{mpp}.
-\\ See Section~\ref{dep:mpp_header}.
+\ChangelogRef{dep:mpp_header}%
 %
 \item Deprecated the \FUNC{shmem\_wait} functions and the \CTYPE{long}-typed \CorCpp \FUNC{shmem\_wait\_until} function.
-\\ See Section \ref{subsec:p2p_intro}.
+\ChangelogRef{subsec:shmem_wait_until, dep:shmem_wait, dep:long_typed_shmem_wait_until}%
 %
 \item Added the \FUNC{shmem\_test} functions.
-\\ See Section \ref{subsec:p2p_intro}.
+\ChangelogRef{subsec:p2p_intro}%
 %
 \item Added the \FUNC{shmem\_calloc} function.
-\\ See Section \ref{subsec:shmem_calloc}.
+\ChangelogRef{subsec:shmem_calloc}%
 %
 \item Introduced the thread safe semantics that define the interaction between
     \openshmem routines and user threads.
-\\See Section \ref{subsec:thread_support}.
+\ChangelogRef{subsec:thread_support}%
 %
 \item Added the new routine \FUNC{shmem\_init\_thread} to initialize the
     \openshmem library with one of the defined thread levels.
-\\See Section \ref{subsec:shmem_init_thread}.
+\ChangelogRef{subsec:shmem_init_thread}%
 %
 \item Added the new routine \FUNC{shmem\_query\_thread} to query the thread
     level provided by the \openshmem implementation.
-\\See Section \ref{subsec:shmem_query_thread}.
+\ChangelogRef{subsec:shmem_query_thread}%
 %
 \item Clarified the semantics of \FUNC{shmem\_quiet} for a multithreaded
     \openshmem \ac{PE}.
-\\See Section \ref{subsec:shmem_quiet}
+\ChangelogRef{subsec:shmem_quiet}%
 %
 \item Revised the description of \FUNC{shmem\_barrier\_all} for a multithreaded
     \openshmem \ac{PE}.
-\\See Section \ref{subsec:shmem_barrier_all}
+\ChangelogRef{subsec:shmem_barrier_all}%
 %
 \item Revised the description of \FUNC{shmem\_wait} for a multithreaded
     \openshmem \ac{PE}.
-\\See Section \ref{subsec:shmem_wait_until}
+\ChangelogRef{subsec:shmem_wait_until}%
 %
 \item Clarified description for \CONST{SHMEM\_VENDOR\_STRING}.
-\\See Section \ref{subsec:library_constants}.
+\ChangelogRef{subsec:library_constants}%
 %
 \item Clarified description for \CONST{SHMEM\_MAX\_NAME\_LEN}.
-\\See Section \ref{subsec:library_constants}.
+\ChangelogRef{subsec:library_constants}%
 %
 \item Clarified API description for \FUNC{shmem\_info\_get\_name}.
-\\See Section \ref{subsec:shmem_info_get_name}.
+\ChangelogRef{subsec:shmem_info_get_name}%
 %
 \item Expanded the type support for RMA, AMO, and point-to-point
     synchronization operations.
-\\ See Tables \ref{stdrmatypes}, \ref{stdamotypes}, \ref{extamotypes}, and
-    \ref{p2psynctypes}
+\ChangelogRef{stdrmatypes,, stdamotypes,, extamotypes,, p2psynctypes}%
```

Comment:
Why `,,`?

Reply:
These are Table references. By default, cleveref will sort and compress the references, but you can use `,,` to separate the list items to tell cleveref to *not* compress. The effect will be something akin to:
`See Tables 1, 2, 3 and 4.`
instead of
`See Tables 1 to 4.`
I could have done it either way, but since I felt that the Table hyperref links were useful, I didn't want to compress them away.

Reply:
I will add a note here as a reminder.

Reply:
Thanks!